R/print.R

```diff
@@ -23,10 +23,18 @@ print.equation <- function(x, ...) {
 #' @param ... not used
 #'
 #' @method knit_print equation
+#'
+#' @importFrom knitr knit_print asis_output
+#' @importFrom texPreview tex_preview
 #' @noRd
-
-knit_print.equation <- function(x,...){
-  knitr::asis_output(format(x))
+#'
+knit_print.equation <- function(x,...,tex_packages = "\\renewcommand*\\familydefault{\\rmdefault}"){
+  eq <- format(x)
+  if(isTRUE(knitr::opts_knit$get("rmarkdown.pandoc.to") == "gfm")){
+    knit_print(tex_preview(eq, usrPackages = tex_packages))
```

Comment:
I'd prefer if we could keep `texPreview::tex_preview` as a suggests. Can we add this function

```r
#' @keywords internal
#' @noRd
is_texPreview_installed <- function() {
  requireNamespace("texPreview", quietly = TRUE)
}
```

And then add a conditional to this section of code so it looks like this?

```r
if (!is_texPreview_installed()) {
  message("Please install \"{texPreview}\" with `install.packages(\"texPreview\")` for equations to render with GitHub flavored markdown. Defaulting to raw TeX code.", call. = FALSE)
  print(eq)
} else {
  knit_print(texPreview::tex_preview(eq, usrPackages = tex_packages))
}
```

Reply:
sure thing. Would you like to make the update, or I can this evening.

Reply:
Yeah if you wouldn't mind making those changes that would be great!

Reply:
Applied :)